### PR TITLE
Fix deprecation warnings.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: tests/TestCase/Panel/VariablesPanelTest.php [hunk 1]
```diff
@@ -74,9 +74,9 @@ public function testShutdown()
 
         $unserializableDebugInfo = $this
             ->getMockBuilder('\stdClass')
-            ->setMethods(['__debugInfo'])
+            ->addMethods(['__debugInfo'])
             ->getMock();
-        $unserializableDebugInfo->expects($this->once())->method('__debugInfo')->willReturn([
+        $unserializableDebugInfo->expects($this->any())->method('__debugInfo')->willReturn([
```
Comment (ADmad) [inline tests/TestCase/Panel/VariablesPanelTest.php:79]: @markstory This test was failing as `__debugInfo()` was being called more than once. I couldn't figure out how/why that was happening. Most likely related to the Debugger internals changes.

**Comment by ADmad** [comment]
The remaining failures are probably due to tables not being dropped/truncated between tests with the new fixture manager.

File: tests/TestCase/Panel/SqlLogPanelTest.php [hunk 2]
```diff
@@ -25,13 +25,6 @@
  */
 class SqlLogPanelTest extends TestCase
 {
-    /**
-     * fixtures.
-     *
-     * @var array
-     */
-    public $fixtures = ['core.Articles'];
```
Comment (markstory) [inline tests/TestCase/Panel/SqlLogPanelTest.php:33]: Do we need the records in this fixture?

File: tests/TestCase/Panel/VariablesPanelTest.php (hunk 1 above)
Comment (markstory) [reply tests/TestCase/Panel/VariablesPanelTest.php:77]: I suspect you're right. Having the mock always return the same value for __debugInfo should be fine.

File: tests/schema.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+declare(strict_types=1);
+
+use Cake\Database\Schema\TableSchema;
+
+/**
+ * Abstract schema for Debugkit tests.
+ */
+return [
+    [
+        'table' => 'panels',
```
Comment (markstory) [inline tests/schema.php:11]: I wonder if we should also update the schema generation tricks we use in DebugKit to build the database at runtime to also use this schema format.

**Review by markstory — COMMENTED** [review]
Thanks for getting this started. I can take care of getting the rest of the tests passing if you'd like.

**Comment by ADmad** [comment]
> I can take care of getting the rest of the tests passing if you'd like.

Yes please.

File: tests/TestCase/Panel/SqlLogPanelTest.php (hunk 2 above)
Comment (ADmad) [reply tests/TestCase/Panel/SqlLogPanelTest.php:33]: Yes there are tests using Articles table/fixture.

**Pull request merged**